internal/app/export/service.go

```diff
@@ -109,20 +109,25 @@ func (e *exporter) Start(queue string) error {
 	return nil
 }
 
-func (e *exporter) Publish(subject, topic string, payload []byte) {
+func (e *exporter) Publish(subject, topic string, payload []byte) errors.Error {
 	e.logger.Debug(fmt.Sprintf("Publishing to topic %s", topic))
 	if err := e.publish(topic, payload); err == nil {
```

Comment:
Please turn the logic here - use `if err != nil`

Reply:
this is more elegant because it has to exit if there is no error
if there is an error it should continue and if there is cache add the message to cache